user-interface/src/main/resources/application.properties

```
@@ -109,3 +109,6 @@ spring.mustache.check-template-location=false
 #EHCache used by spring security ACL to cache ACEs
 spring.cache.jcache.config=classpath:ehcache3.xml
 ###############################################################################
+################### Token Generation ##########################################
+qbic.access-token.salt=${ACCESS_TOKEN_SALT:}
```

What is the reason for the salt itself ending on a colon? 
```suggestion
qbic.access-token.salt=${ACCESS_TOKEN_SALT:}
```

---

The colon separates the value from the default. No default value exists here so the default will be empty (which is checked in the code)